src/main/cpp/t04_both2.cpp

```diff
@@ -25,7 +25,70 @@
 #include <iostream>
 
 using namespace std;
-
+void swap(int *a, int *b)
+{
+    int temp = *a;
+    *a = *b;
+    *b = temp;
+}
 int t04_both2() {
+  int n,m,brN=0,k;
+  bool flag=false;
+  cin>>n;
+  int a[n];
+  for(int i=0;i<n;i++)
+  {
+    cin>>k;
+    for(int j=0;j<i;j++)
+    {
+      if(k==a[j]){flag=true;break;}
+
+    }
+    if(!flag){a[brN]=k;brN++;}
+    flag=false;
+  }
+  cin>>m;
+  int b[m];
+  int brM=0;
+  for(int i=0;i<m;i++)
+  {
+    cin>>k;
+    for(int j=0;j<i;j++)
+    {
+      if(k==b[j]){flag=true;break;}
+
+    }
+    if(!flag){b[brM]=k;brM++;}
+    flag=false;
+  }
+  int br=0;
 
+  if(brN>brM)k=brM;
+  else k=brN;
+  int c[k];
+  for(int i=0;i<brN;i++)
+  {
+    for(int j=0;j<brM;j++)
+    {
+      if(a[i]==b[j]){c[br]=a[i];br++;break;}
+    }
+  }
+   bool swapped;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'swapped' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4586365)